config/samples/image_cache/README.md

```diff
@@ -72,3 +72,107 @@ that periodically executes the `glance-cache-cleaner` utility.
 
 You can find more about image-cache configuration options in the
 [upstream](https://docs.openstack.org/glance/latest/admin/cache.html) documentation.
+
+## How to test
+
+Assuming a given `GlanceAPI` instance has been scaled up, it is possible to
+perform cache related operations and verify that `Pods` are able to see each
+other (or resolve them by `hostname`) and the image can be cached on either
```

Comment:
can we explain here that a caching operation is related to a specific replica (through the `--os-image-url` parameter), and that we must require a `backend != File`, otherwise it will be stuck in `queued` state?
Is a requirement that Pods are able to see each other? I thought that was something to help us identifying the replica from the cli.
If my last sentence is true, do we have an API to list the replicas / glanceAPI instances that are part of the same deployment? 
This might be a good idea for an upstream RFE, so that people who want to use cache, can list the available instances (replicas of the same API).

Reply:
No requirement of Pods to see each other.
We need a new RFE to list down available set of glance API service(s) running in local environment.